flux-middle/src/fhir.rs

```diff
@@ -407,6 +407,18 @@ newtype_index! {
     pub struct SortVid {}
 }
 
+#[derive(Clone, PartialEq, Eq, Hash, TyEncodable, TyDecodable)]
+pub enum SortCtor {
+    Set,
+    User { name: Symbol, arity: usize },
+}
+
+impl SortCtor {
+    pub fn set() -> Self {
+        SortCtor::Set
+    }
+}
```

Comment:
I'd remove this, writing `SortCtor::Set` sounds better than `SortCtor::set()`

```suggestion
```

Reply:
done!